### PR TITLE
CHUI-49: Add alert in case of error during place order

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What problem is this solving?

Adds an error alert when any request fail during the place order flow.

#### How should this be manually tested?

[Workspace](https://placefail--checkoutio.myvtex.com/cart/add?sku=289).

Test plan:

1. Enter the above workspace.
2. Go to checkout.
3. Enter a second purchase email.
4. Select payment with a new credit card and fill the required inputs.
5. In the review step, reload the page.
6. Press the place order button, you should see the error alert.

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [x] Linked this PR to a Jira story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: messages/en.json [hunk 1]
```diff
@@ -2,5 +2,7 @@
   "store/place-order-button": "Place Order",
   "store/checkout-product-count": "{quantity, plural, one {# product} other {# products}}",
   "store/edit-cart-label": "Edit Cart",
-  "store/view-cart-label": "View Cart"
+  "store/view-cart-label": "View Cart",
+  "store/place-order.close-error-alert-label": "Close",
+  "store/place-order.generic-error-message": "The transaction can't be completed. Please try again soon."
```
Comment (marcoskwkm) [inline messages/en.json:7]: Shouldn't this be in past tense?

**Review by marcoskwkm — APPROVED** [review]

File: messages/en.json (hunk 1 above)
Comment (lucasecdb) [reply messages/en.json:7]: I "extracted" this message directly from figma 😅. you suggest to change it to something like the following?

```suggestion
  "store/place-order.generic-error-message": "The transaction couldn't be completed. Please try again soon."
```

Comment (marcoskwkm) [reply messages/en.json:7]: Yes, I _think_ that sounds a bit more natural, but if that's how it is on Figma then I we can keep it like this I guess.

Comment (lucasecdb) [reply messages/en.json:7]: we could ask Vlad to review the english translations as well when the time comes for the translations reviews

**Review by darlenedms — APPROVED** [review]

**Pull request merged**